samples/client/petstore/rust/examples/error_handling.rs

```diff
@@ -0,0 +1,49 @@
+extern crate futures;
```

Comment:
The other example isn't generated, it just lives in the examples folder.

Since these are petstore-sample specific, it seems right to not generate them.